File: golang/arrayandhashing/containsDuplicate.go
package arrayandhashing

func ContainsDuplicate(nums []int) bool {
	Hashset := make(map[int]bool)
	for _, v := range nums {
		if Hashset[v] {
			return true
		}
		Hashset[v] = true
	}
	return false
}

File: javascript/containsDuplicate.js
function containsDuplicate(nums) {
  const hashset = new Set();
  for (let i = 0; i < nums.length; i++) {
    if (hashset.has(nums[i])) {
      return true;
    }
    hashset.add(nums[i]);
  }
  return false;
}

console.log(containsDuplicate([1, 2, 3, 1]));
